### ISSUE TITLE
Clarification: Are multimodal LLM checkpoints allowed for the Playschool Challenge?

### ISSUE THREAD

**Issue by osamanatouf2** [lm-playpen/playpen#28]
wanted to ask for clarification about whether multimodal LLM checkpoints are allowed for the LM Playschool Challenge.

I am considering using a model such as Qwen/Qwen2.5-VL-7B-Instruct. This is a vision-language model, but my intended use would be text-only: the model would receive only dialogue/task text as input, with no images or visual inputs during training or evaluation.

The reason I am asking is that this checkpoint is not loaded as a normal text-only causal language model. For example, loading it with:

AutoModelForCausalLM.from_pretrained("Qwen/Qwen2.5-VL-7B-Instruct")

raises an error because the model configuration is Qwen2_5_VLConfig, not a standard text-only causal LM config. It has to be loaded with something like:

Qwen2_5_VLForConditionalGeneration.from_pretrained(...)

So even if I only use the text/dialogue interface, the base checkpoint itself is still multimodal.

Could you clarify the following?

Are multimodal pretrained checkpoints allowed if we only use text inputs and outputs for the challenge?
Is it allowed to fine-tune a multimodal LLM on dialogue interaction data for the Playschool Challenge?
If the benchmark provides only text-based observations/actions, is there any restriction on starting from a vision-language checkpoint?
Would using a multimodal checkpoint be considered outside the intended rules or an unfair advantage compared to using a text-only LLM?

I am mainly interested in adapting reasoning/training ideas from multimodal LLMs to goal-oriented dialogue interaction, but I want to make sure the base model choice is allowed before proceeding.

Thanks!
